driver/src/main/com/mongodb/ListIndexesIterableImpl.java

```diff
@@ -33,9 +33,9 @@
     private final CodecRegistry codecRegistry;
     private long maxTimeMS;
 
-    ListIndexesIterableImpl(final MongoNamespace namespace, final Class<TResult> resultClass, final CodecRegistry codecRegistry,
-                            final ReadPreference readPreference, final OperationExecutor executor) {
-        super(executor, ReadConcern.DEFAULT, readPreference); // TODO: ReadConcern?
+    ListIndexesIterableImpl(final ClientSession clientSession, final MongoNamespace namespace, final Class<TResult> resultClass,
+                            final CodecRegistry codecRegistry, final ReadPreference readPreference, final OperationExecutor executor) {
+        super(clientSession, executor, ReadConcern.DEFAULT, readPreference); // TODO: ReadConcern?
```

Comment:
Rather than a todo - does this warrant a spec ticket?

Reply:
Don't think it warrants a spec ticket.  I'll remove the TODO.